### ISSUE TITLE
Short SHA missing from version information

### ISSUE THREAD

**Issue by abanbj** [GenericMappingTools/gmt#352]
This may be a deliberate thing, but if not;

The short hash that is appended to the version number (``6.0.0_[short SHA]``) is gone. I'm using d0302c4.
```
$ gmt --version
6.0.0
$ gmt

          GMT - The Generic Mapping Tools, Version 6.0.0 [64-bit]
(...)
(...)
```

**Comment by abanbj** [comment]
Any news on this?

**Comment by joa-quim** [comment]
Yes, it's fixed for some time now.

**Comment by abanbj** [comment]
Hmm, I'm still`missing the hash
```
$ gmt --version
6.0.0
``

**Comment by abanbj** [comment]
OK, I wiped my Winbuntu installation and started fresh. Now it shows. Do not understand what causes this.

**Comment by KristofKoch** [comment]
Mh, I still have this problem … maybe some debug/dev options need to be activated in  `ConfigUser.cmake`?

```
kristof@h2813222:~$ gmt --version
6.0.0
kristof@h2813222:~$ cd /usr/local/src/gmt/
kristof@h2813222:/usr/local/src/gmt$ git branch -l
* nicerectclip
kristof@h2813222:/usr/local/src/gmt$
```

**Comment by abanbj** [comment]
Interesting @KristofKoch. The SHA disappeared when ~something was changed. So why wouldnt it reappear again when it was changed back?

**Comment by abanbj** [comment]
And now it's gone again!

```
$ gmt --version
6.0.0
```

**Comment by KristofKoch** [comment]
Well, I can report it works here now …

```
kristof@h2813222:/usr/local/src/gmt$ gmt --version
6.0.0_f0ec063
```

Why? I have no clue. I did not touch `ConfigUser.cmake`, just `git pull` and build it.

**Comment by abanbj** [comment]
Reopening this.

**Comment by KristofKoch** [comment]
Currently works here on Ubuntu 18.04.1 LTS (GNU/Linux 4.15.0 x86_64)

```
kristof@h2813222:~$ gmt --version
6.0.0_f0ec063
```

**Comment by abanbj** [comment]
Fixed by #447.
